conventions/documentation/quick-starts.md

```diff
@@ -0,0 +1,205 @@
+
+![buttons](../images/qs-buttons.png)
+
+#### Check your work
+Check your work is intended to help the user validate that they have correctly followed the steps in a task. They typically ask the user to validate that the task is complete.
+
+If a user clicks **Yes**, the feature will turn green. If a user clicks **No**, the feature will turn red and an error message will be shown.
+
+In the future, we want to explore how we can show more specific troubleshooting help if users choose **No**.
+
+Whichever state the Check your work feature is in (neutral – blue, yes – green, or no – red) will be reflected in the task header’s color and icon when users click Next.
+
+![check your work](../images/qs-checkyourwork.png)
+
+#### Final screen buttons
+The last screen should have a primary **Close** button, a secondary **Back** button, and a link to **View all quick starts**. If applicable, also show a link to the next quick start in the “bundle”.
+
+![final screen buttons](../images/qs-finalscreenbuttons.png)
+
+#### Hints (4.7+)
+When the user is in the quick start and the quick start mentions a location in the vertical navigation or masthead, the navigation point will look like a link and the user can click on it to reveal the hint animation. The animation would go through one cycle and then stop. A solid blue rectangle would remain around the navigation point after the animation stops or until the user moves onto the next step in the quick start or quits the quick start.
+
+If the user clicks on a sub navigation point hint in a quick start, the main navigation section should be highlighted.
+
+![hint](../images/qs-hint.png)
+
+## Part II: What kind of content do I need to contribute a quick start?
+All items are required unless otherwise specified.
+
+### Catalog tile
+Each quick start will have a tile that users can click to open the quick start.
+![catalog tile](../images/qs-tile.png)
+1. **Quick start name:** Each quick start should have a straightforward and succinct name.
+2. **Icon / image:** Quick starts should have an icon or other image to surface on the quick starts catalog page. If no icon is provided, a default quick starts icon will be used.
+3. **Description:** A one or two sentence description should be available to surface on the quick start tiles that appear on the quick starts catalog page.
+4. **Time estimate:** Quick starts should have a time estimate that will be shown on the quick start tile and in the heading of the side panel. Quick starts should typically be no longer than 20-30 minutes.
+5. **Badges:** Quick start tiles can have a number of badges. Each quick start should have the time estimate in a badge. If a quick start has been started or completed, show the relevant badge: In progress or Completed.
+6. **Prerequisites (Optional):** When applicable, detail what prerequisites users / clusters have to meet before completing the quick start. The prerequisites should appear underneath the description.
```

Comment:
Is there a limit on the prerequisites? We could see a very tall card if one quick start has more content than the others. Should we have a design for when there is more content than that card can manage?

Reply:
Should this design show a prerequisite to correspond to the number?

Reply:
Regarding the Prerequisites - in Virtualization we have a QS that we want to refer users to documentation on how to configure the bridges on the nodes before creating a NAD. It is part of the prerequisites explanation and I was wondering if that link to the documentation should appear on the tile itself or on the right panel.